bundle/manifests/netobserv-operator.clusterserviceversion.yaml

```diff
@@ -813,7 +813,7 @@ spec:
     - apiGroups:
       - netobserv.io
       apiVersions:
-      - v1alpha1
+      - v1beta1
```

Comment:
Just a thought, but shouldn't we specify here both v1alpha1 and v1beta1 ?

Reply:
@jotak AFAIK we define only the latest version here , however in the conversion webhook we define both